### PR TITLE
Adds disclaimer to not use this list an excuse to stay away from Kubernetes

### PR DESCRIPTION
**Pull request by kamranahmedse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,10 @@ learn from others and reduce the unknown unknowns of running Kubernetes in produ
 For more information, [see the blog post](https://srcco.de/posts/kubernetes-failure-stories.html).
 
 
+# Disclaimer
+
+Please note that the purpose of this repository is to not denote that Kubernetes is bad and you should just stay away from it. The idea is to help you learn from these postmortems and mistakes of others and to not repeat them. Similar to this repository, there could have been a list of "Encryption Failure Stories" but that doesn't mean we shouldn't encrypting things – so don't use this repository as an excuse to stay away from Kubernetes.
```
Comment (romoy) [inline README.md]: What about rephrasing it like the following to promote the intent? 

```suggestion
Please note that the purpose of this repository is to learn from Kubernetes postmortems. These postmortems provide the experiences of your peers as potential guides to you on how you could troubleshoot and enhance your Kubernetes infrastructure.
```

**Comment by hjacobs** [comment]
:laughing: see also my blog post https://srcco.de/posts/why-kubernetes.html

**Comment by hjacobs** [comment]
@kamranahmedse thanks for the PR, let me think about the text --- I obviously was never intending to make Kubernetes "look bad". Information can always be misused, so I fully see that some orgs might point people to this list and say "see, that's why you should not use K8s" --- you can never fully prevent that.

**Comment by andrerpena** [comment]
image

**Comment by andrerpena** [comment]
image

**Comment by kamranahmedse** [comment]
Yeah, I agree. Anyways, up to you :)

**Pull request closed without merging**